README.md

```diff
@@ -2,16 +2,33 @@
 
 [![Build Status](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt.svg?branch=master)](https://travis-ci.org/sonatype-nexus-community/nexus-repository-apt) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-apt/depshield.svg)](https://depshield.github.io)
 
+> **WARNING: 3.17.0 includes a feature flagged version of the APT plugin by default**. We're releasing this way so that 
+we can get early feedback on the APT plugin without impacting the wider user base. From 3.18.0 APT will be 
+enabled by default.
+
+>If you are an APT user upgrading to 3.17 setting ```nexus.apt.enabled=true``` in ```./etc/nexus.properties``` will 
+enable the plugin. **You will not be able to install the community plugin.**
```

Comment:
Q: Otherwise no change is needed and their data will be intact?  If so may be worth saying that.

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-apt/pull/116/commits/53c1a11634be6bc058c71a103e364e5bfa42137d